config/common/group_vars/k8s-cluster/ck8s-k8s-cluster.yaml

```diff
@@ -0,0 +1,89 @@
+kube_apiserver_enable_admission_plugins:
+  - "PodSecurityPolicy"
+  - "NamespaceLifecycle"
+  - "LimitRanger"
+  - "ServiceAccount"
+  - "DefaultStorageClass"
+  - "DefaultTolerationSeconds"
+  - "MutatingAdmissionWebhook"
+  - "ValidatingAdmissionWebhook"
+  - "ResourceQuota"
+  - "NodeRestriction"
+
+kube_kubeadm_apiserver_extra_args:
+  delete-collection-workers: "3"
+
+kube_proxy_mode: iptables
+kube_proxy_metrics_bind_address: "0.0.0.0:10249"
+
+enable_nodelocaldns: false
+
+kubernetes_audit: true
+audit_log_path: "/var/log/kube-audit/kube-apiserver.log"
```

Comment:
Should we perhaps use the default value `/var/log/audit/kube-apiserver-audit.log`?
I know it will require changes to fluentd but maybe it is worth it to be as vanilla as possible.